lab2/MySQL_DB_Select.sql

```diff
@@ -0,0 +1,546 @@
+create database if not exists Labor_SQL;
+use Labor_SQL;
+
+CREATE TABLE Laptop (
+	code int NOT NULL ,
+	model varchar (50) NOT NULL ,
+	speed smallint NOT NULL ,
+	ram smallint NOT NULL ,
+	hd real NOT NULL ,
+	price decimal(8,2) NULL ,
+	screen tinyint NOT NULL 
+);
+
+CREATE TABLE PC (
+	code int NOT NULL ,
+	model varchar (50) NOT NULL ,
+	speed smallint NOT NULL ,
+	ram smallint NOT NULL ,
+	hd real NOT NULL ,
+	cd varchar (10) NOT NULL ,
+	price decimal(8,2) NULL 
+);
+
+CREATE TABLE Product (
+	maker varchar (10) NOT NULL ,
+	model varchar (50) NOT NULL ,
+	type varchar (50) NOT NULL 
+);
+
+CREATE TABLE Printer (
+	code int NOT NULL ,
+	model varchar (50) NOT NULL ,
+	color char (1) NOT NULL ,
+	type varchar (10) NOT NULL ,
+	price decimal(8,2) NULL 
+);
+
+ALTER TABLE Laptop ADD 
+	CONSTRAINT PK_Laptop PRIMARY KEY  NONCLUSTERED 
+	(
+		code
+	);
+
+ALTER TABLE PC  ADD 
+	CONSTRAINT PK_pc PRIMARY KEY  NONCLUSTERED 
+	(
+		code
+	);
+
+ALTER TABLE Product  ADD 
+	CONSTRAINT PK_product PRIMARY KEY  NONCLUSTERED 
+	(
+		model
+	); 
+
+
+ALTER TABLE Printer  ADD 
+	CONSTRAINT PK_printer PRIMARY KEY  NONCLUSTERED 
+	(
+		code
+	); 
+
+ALTER TABLE Laptop ADD 
+	CONSTRAINT FK_Laptop_product FOREIGN KEY 
+	(
+		model
+	) REFERENCES Product (
+		model
+	);
+
+ALTER TABLE PC ADD 
+	CONSTRAINT FK_pc_product FOREIGN KEY 
+	(
+		model
+	) REFERENCES Product (
+		model
+	);
+
+ALTER TABLE Printer ADD 
+	CONSTRAINT FK_printer_product FOREIGN KEY 
+	(
+		model
+	) REFERENCES Product (
+		model
+	);
+
+#---Product------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
+insert into Product values('B','1121','PC');
```

Comment:
you don't need to duplicate it, you can write it in this way ('B','1121','PC'), ('A','1232','PC')